test/replicates.js

```diff
@@ -111,3 +113,86 @@ tape('replicates empty directories', function (t) {
     stream.pipe(streamClone).pipe(stream)
   })
 })
+
+tape('downloads empty files to fs', function (t) {
+  var drive = hyperdrive(memdb())
+  var driveClone = hyperdrive(memdb())
+  var tmp = path.join(os.tmpdir(), 'hyperdrive-test')
+  mkdirp.sync(tmp)
+
+  var archive = drive.createArchive()
+
+  var ws = archive.createFileWriteStream('empty.txt')
+  ws.end()
```

Comment:
can be refactored to `archive.createFileWriteStream('empty.txt').end()`. i was looking where else `ws` was used because it got a name